### ISSUE TITLE
sync function and menu with the official main site

### ISSUE THREAD

**Issue by Boytobeaman** [Summer-andy/topology-react#1]
In this react version, some function and menu are missing comparing with the official site

For example:
/workspace page:

In the right section:
we may need to have :外观，事件，动效等功能
![image](https://user-images.githubusercontent.com/16664656/96535481-7e7ddd80-12c4-11eb-9b0f-31fa0c069528.png)

In the left section:
we may need to have :系统组件，我的组件，我的图片等
![image](https://user-images.githubusercontent.com/16664656/96536312-57c0a680-12c6-11eb-9554-c45d434d4749.png)

To summarize：we may need to sync menu and function with the offical site
